source/install/requirements.rst

```diff
@@ -26,12 +26,21 @@ Client Software
 PC Web Experience
 ^^^^^^^^^^^^^^^^^
 
--  PC: Windows 7, Windows 8, Windows 10 with IE 11*, Chrome 43+, Firefox 52+, and Edge 40+ (or EdgeHTML v15+)
--  Mac: OS 10 (Safari 9, Chrome 43+)
+-  PC: Windows 7+ with IE 11*, Chrome 43+, Firefox 52+, and Edge 40+ (or EdgeHTML v15+)
+-  Mac: OS 10.8+ (Safari 9, Chrome 43+)
 -  Linux: Arch 4.0.0 (Chrome 43+)
```

Comment:
I know i have mentioned this before. But Arch doesn't really have versions. We can just drop Arch and just say Linux: (Chrome 59+ Firefox 60+)

Reply:
I can drop Arch, no problem